invenio_rdm_records/services/config.py

```diff
@@ -113,6 +115,18 @@ class RDMSearchVersionsOptions(SearchVersionsOptions, SearchOptionsMixin):
     """Search options for record versioning search."""
 
 
+class RDMRecordCommunitiesConfig(ServiceConfig, ConfiguratorMixin):
```

Comment:
if we go with this independent service:
1. you might want to break the inheritance `class RecordCommunitiesService(RecordService):` and do not depend on `RecordService`
2. you might want to remove the 
```
@property
def record_communities
```
in the main service

Reply:
We will need another proxy, `current_<name>_service`

Reply:
Not sure we need that. In the resource we use the service, I am not sure we should use `current_<name>_service` there , (see resource [here](https://github.com/inveniosoftware/invenio-rdm-records/pull/1178/files#diff-a78ef7917e10ab1669945ba1a0f2ba7a46ac560e5b1d14e9c3875878b2f5c433R170))

Maybe I am missing something ...

Reply:
I agree with decoupling but also using `current_...` in the resources breaks the design of `resources->service`. I'm interested in the outcome of this one. I'm not sure what's best.

Reply:
Discussed IRL, decided to go for independent service.